source/yellow/index.html

```diff
@@ -203,8 +203,8 @@ <h1>Pick your flavor</h1>
 <div class="material-card text spaced-card">
   <h1>Open hardware</h1>
   <p>
-    We spend a lot of time and resources on getting our products to work great. We want to make it possible for other people to study our work and be able to learn how to make great products for the smart home, so we've released the schematics of Yellow as open-source.
-    <a href="https://yellow.home-assistant.io/yellow-v1.2-standard/">Explore the board</a> and <a href="https://github.com/NabuCasa/yellow/">view the source.</a>
+    We spend a lot of time and resources on fine-tuning our products. We want other people to be able to study our work and be able to make their own great smart home products. For this reason, we've released the schematics of Home Assistant Yellow as open-source.
+    <a href="https://yellow.home-assistant.io/hardware/yellow-v1.3-standard/">Explore the board</a> and <a href="https://github.com/NabuCasa/yellow/">view the source.</a>
```

Comment:
Let's use the PoE version here, the Standard looks so empty.

```suggestion
    <a href="https://yellow.home-assistant.io/hardware/yellow-v1.3-poe/">Explore the board</a> and <a href="https://github.com/NabuCasa/yellow/">view the source.</a>
```